src/renderer/colors.css

```diff
@@ -0,0 +1,53 @@
+:root {
+    /* color palette */
+    --gray-50: #f7fafc;
+    --gray-100: #edf2f7;
+    --gray-200: #e2e8f0;
+    --gray-300: #cbd5e0;
+    --gray-400: #a0aec0;
+    --gray-500: #718096;
+    --gray-600: #4a5568;
+    --gray-650: #3e485a;
+    --gray-700: #2d3748;
+    --gray-750: #252d3c;
+    --gray-800: #1a202c;
+    --gray-850: #191d28;
```

Comment:
I had to hard-code the interpolation between 800 and 900. I'm thinking about introducing SCSS, so I can use [`mix`](https://sass-lang.com/documentation/modules/color#mix).